bldsys/cmake/shader_helper.cmake

```diff
@@ -0,0 +1,104 @@
+add_custom_target(vkb__shaders)
+
```

Comment:
```suggestion

find_program(DXC_EXECUTABLE dxc HINTS $ENV{VULKAN_SDK}/bin)
if(NOT DXC_EXECUTABLE)
    message(WARNING "dxc not found! Shaders with .hlsl extension will not be compiled to SPIR-V. Please set VULKAN_SDK to the Vulkan SDK path.")
endif()
find_program(SPV_VALIDATOR_EXECUTABLE spirv-val HINTS $ENV{VULKAN_SDK}/bin)
if(NOT SPV_VALIDATOR_EXECUTABLE)
    message(WARNING "spirv-val not found! SPIR-V validation will not be performed. Please set VULKAN_SDK to the Vulkan SDK path.")
endif()
find_program(GLSLC_EXECUTABLE glslc HINTS $ENV{VULKAN_SDK}/bin)
if(NOT GLSLC_EXECUTABLE)
    message(WARNING "glslc not found! Shaders with .glsl extension will not be compiled to SPIR-V. Please set VULKAN_SDK to the Vulkan SDK path.")
endif()

```

Reply:
This might need to be moved to the root CMakeLists.txt or something.  I'm not sure how the scope will work for these values, but there's no point in checking repeatedly.